CONTRIBUTING.md

```diff
@@ -1,98 +1,211 @@
+
+If you have examples for other IaC technologies we'd love to add them too! Please let us know on Slack or GitHub.
+
+
+<p align="center">
+    <img alt="You have to live by my rules" src="https://media.giphy.com/media/ZXffEBBmsI3md367PN/giphy.gif" />
+</p>
+
+Create your core rule file by duplicating an existing rule and renaming it appropriately. Remove all logic from the function at the end for now.
+
+Fill out the `rules.Rule` struct with appropriate information. The fields are described below:
+
+| Field            | Description |
+| ---------------- | ----------- |
+| AVDID            | This is a unique ID that identifies the rule not just within DefSec, but within the [AVD](https://avd.aquasec.com/). The ID is composed of the prefix `AVD-`, three letters representing the provider in capitals e.g. `AWS`, another `-`, and then a 4-digit, zero-padded number. For example: `AVD-AWS-0086`. The easiest way to assign a new AVD ID is to run `grep -r "AVD-" . | grep AVDID | awk -F'"' '{print $2}' | sort -u` to find the highest number for your provider and increment it by one.
+| Provider         | You can set this using a constant from the `provider` package, listed [here](https://github.com/aquasecurity/defsec/blob/master/provider/provider.go#L8-L21)
+| Service          | A string representing the service your rule relates to (lower-case), e.g. `s3`. This will generally match the package your rule is inside.
+| ShortCode        | This is a human-readable identifier for your check that uniquely describes it within the provider and service. e.g. `block-public-acls`
+| Summary          | A short paragraph that summarises what best-practice the rule is trying to enforce. e.g. *Data stored in service X should be encrypted at rest*
+| Impact           | A short sentence describing the security impact of the rule failing e.g. *All data stored using service Y will be publicly available*.
+| Resolution       | A short sentence describing how to resolve the issue. This shouldn't be tied to implementation technology such as Terraform/CloudFormation, but written at a higher level in terms of Cloud Resources and configuration e.g. *Enable encryption for this bucket*
+| Explanation      | A detailed explanation of why this is an issue. Often very short for simple rules.
+| Links            | A list of external references/guides that relate to the rule.
+| Severity         | The severity of the check.
+| Terraform/CloudFormation | These contain good and bad code examples (those that pass and fail the rule respectively); a list of IaC technology specific links e.g. a link to the specific resource documentation on the [Terraform Registry](https://registry.terraform.io/); and a markdown block that describes remediation using the specific IaC technology e.g. *Set the Terraform parameter `enable_encryption` to `true`*
+
+We currently use the following guide to approximate severity:
```

Comment:
```suggestion
We use the following guide to approximate severity:
```
I use the word `currently` way too often so its jarring. It reminds me of a holiday in Mallorca where the Rep said "Currently the capital is Palma"....... NO, the capital IS Palma